packages/connect-form/src/components/advanced-options-tabs/ssh-tunnel-tab/proxy-and-ssh-tunnel-tab.spec.tsx

```diff
@@ -0,0 +1,269 @@
+import React from 'react';
+import { render, screen, fireEvent, cleanup } from '@testing-library/react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import ConnectionStringUrl from 'mongodb-connection-string-url';
+import { ConnectionOptions } from 'mongodb-data-service';
+
+import ProxyAndSshTunnelTab from './proxy-and-ssh-tunnel-tab';
+import { ProxyOptions } from 'mongodb';
```

Comment:
optional, feel free to leave out:

```suggestion
import type { ProxyOptions } from 'mongodb';
```

to clarify that we only use the types from the package, not the driver API itself

Reply:
that's actually a good practice. doing it in all the places.